sta_exercise_final.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    def calc_sta(stims, spiks, dt, num_lags):
</pre><p>Logic is completely good here. However, the workflow is such that you will want to produce the "design matrix" X (matrix with lagged stimulus) and separately produce the R. Once these are produced, you can use them for many different things. For example, the STA is just ~ X@R, and the filtered stimulus is then X@sta. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuroTheoryUMD/mattjac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>